scripts/canbus.sh

```diff
@@ -23,6 +23,11 @@ cd "${DIR}/.."
 
 source "$DIR/apollo_base.sh"
 
+MACHINE_ARCH=$(uname -m)
```

Comment:
Put this in scripts/apollo_base.sh: setup_device function so that we can manage all devices at the same place.